### ISSUE TITLE
Procedure for merging compositions

### ISSUE THREAD

**Issue by mscuthbert** [diamm/diamm#392]
Hello -- I'm wondering if there is a procedure for merging compositions -- e.g., there are a lot more Mass movements by Antonio Zacara da Teramo than he actually wrote -- a lot of different sources for the same piece are listed as different pieces.  Wondering if there's an admin tool to move and merge records for different pieces.  Thanks!

**Comment by mscuthbert** [comment]
To delete once I fully understand procedure:
composition/69670 [dup. of Sus une fontayne -- all relevant information merged.

**Comment by ahankinson** [comment]
If you select two or more compositions in the admin interface you can use the dropdown menu to 'merge' them. It will take the FIRST listed composition as the primary composition, and merge the references, etc. into that one from the others.

But, you should check with @juliacmcf first about this -- I seem to remember there was some strangeness about duplicating item records for merged compositions? At any rate, you should do them one at a time, and then check to make sure it merged correctly.

**Comment by ahankinson** [comment]
If it has no Item record, it won't appear in any sources. They're generally the exception and not the rule.

**Comment by mscuthbert** [comment]
Ah! I was thinking you meant image-compositon-item records not source-composition-item records

**Comment by juliacmcf** [comment]
We may have created composition / item records for things in inventories that we hadn’t yet entered, so avoid deleting anything in case it’s wanted. Merging is fine on the composition table as I know we have a lot of redundant pieces that we didn’t merge because we weren’t sure they were the same piece.

Many thanks for working on the database though!
Julia

**Comment by mscuthbert** [comment]
Thanks -- for some entries that are potentially very confusing, such as a source that has five pieces all called "Patrem omnipotentem [Credo]", I've been adding some distinguishing features.  The ones that are tied to the source (i.e., named in the source), I'm very confident adding, for instance, "Patrem omnipotentem [Credo Cursor]" but for those without any distinguishing features I find it very helpful to put a reference to a well known printed edition "Patrem omnipotentem [Credo PMFC 23.110]" -- is this okay?

**Comment by juliacmcf** [comment]
Yes, that’s great. I put in a lot of the Josquin edition numbers (because I had them), so this is fine. Unless Andrew has a better place to put that! We do have bibliographies for compositions, which is where this should also appear, but that doesn’t help if all you can see is the title, so I’m happy with this.
J
